clientv3/watch.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	EventTypeDelete = mvccpb.DELETE
 	EventTypePut    = mvccpb.PUT
+	EventTypeExpire = mvccpb.EXPIRE
```

Comment:
key cannot expire. there is no ttl key in v3. a lease can expire. the key will be deleted. you will not get a expire event from lease since we do not support watch on lease right now.